src/Configuration/src/ConfigServerBase/ConfigServerConfigurationProvider.cs

```diff
@@ -645,22 +648,20 @@ protected internal virtual async Task<ConfigEnvironment> RemoteLoadAsync(string
         try
         {
             using var response = await _httpClient.SendAsync(request).ConfigureAwait(false);
+
+            // Log status
```

Comment:
I'm not a fan of these comments that describe what the code does. Would you consider deleting them?